keyboards/h87a/config.h

```diff
@@ -0,0 +1,57 @@
+#ifndef CONFIG_H
+#define CONFIG_H
+
+#include "config_common.h"
```

Comment:
This include isn't needed, I believe.

Reply:
done.

Reply:
looks like I need at least

#define CONFIG_H

#include "config_common.h"

in there. The #ifdef and #endif at the end of the sequence were removed and it passed.